kivy_ios/toolchain.py

```diff
@@ -462,7 +459,7 @@ def report_hook(index, blksize, size):
 
         if cwd:
             filename = join(cwd, filename)
-        if exists(filename):
+        with suppress(FileNotFoundError):
             unlink(filename)
```

Comment:
That change can avoid a race condition where the file was existing at the moment of the check and no longer just before the unlink